MapboxGeocoder/MBPlacemark.swift

```diff
@@ -352,6 +364,11 @@ open class Placemark: NSObject, Codable {
         }
         return String(describing: houseNumber)
     }
+    
+    /**
+     An array of locations representing the location a user should navigate to to reach the `Placemark`.
+     */
+    @objc open var routableLocations: [CLLocationCoordinate2D]?
```

Comment:
`routableCoordinates` since it's a list of coordinates?

Reply:
_or_, make it [CLLocation]? so that it also bridges to Objective-C?

Reply:
Yes, let’s use an array of CLLocation or Placemark, for Objective-C compatibility.